### DISCUSSION TITLE
block selection

### DISCUSSION THREAD

**Discussion by camargoguedes** [dbeaver/dbeaver#17471] [Q&A]
Bom dia, 
primeiramente gostaria de dizer que a ferramenta é excelente, uma das melhores que já vi. 
Uma implementação que ajudaria muito e a tornaria muito produtiva é a seleção em bloco, igual o NotePad++ o SQL Server e etc. Pois nestes é possivel efetuar a seleção no Teclado diretamente com o atalho de tecla Alt+Shift.

**Comment by ShadelessFox** [comment]
Hello @camargoguedes,

If I understand you correctly, you can enable block selection mode using the `Alt+Shift+A` shortcut.

Please let me know if your issue solved.

**Comment by camargoguedes** [comment]
yes, that would be it, but today it is only possible to make the selection with the mouse, it would not be possible to implement something similar with Notepad++. Is the selection made directly on the text with no zoom change?

**Comment by ShadelessFox** [comment]
Sorry, I don't quite understand. Can you please clarify?

Do you want to hold `Alt` key while dragging the mouse to perform block selection, just like in Notepad++? It's certainly possible to implement, but, as of now, the only solution is to use the mentioned shortcut.

**Comment by camargoguedes** [comment]
that's right, in notepad++ and other tools I don't need to pick up the mouse, we can use the right, left up and down arrows, but thanks anyway

**Comment by ShadelessFox** [comment]
I'm confused. You **don't** want to use the mouse to perform a block selection? This is totally possible in DBeaver, too.

1. Press `Alt+Shift+A` to enter block selection mode
2. Use arrow keys to extend the selection
3. Press `Alt+Shift+A` again to leave block selection mode

**Comment by camargoguedes** [comment]
no mouse.

I'd like to use the arrow keys

<details>

![Setas](https://user-images.githubusercontent.com/54440358/186684259-f6b07de2-2b5b-462e-bb59-124c5586716a.jpeg)
 
</details>

**Comment by ShadelessFox** [comment]
Okay, then use the following solution https://github.com/dbeaver/dbeaver/issues/17470#issuecomment-1227290908
